olive/model/hf_utils.py

```diff
@@ -75,7 +75,14 @@ class HFModelLoadingArgs(ConfigWithExtraArgs):
             " details for the supported parameters."
         ),
     )
-    # TODO(xiaoyuzhang): seems not working. need to check.
+    # for remote files that require authentication
+    token: Union[bool, str] = Field(
```

Comment:
are there any security concerns if we keep text token?

Reply:
The user can always provide `True` to use the saved token. I provided `str` as an option since this parameter also accepts string. Our examples won't use string but the user can decide to use it if they want.

Reply:
Should I say `True` is recommended in order to not expose the token string in the config?

Reply:
Updated description to recommend `True`.

Reply:
I am ok with True. just have some concerns for text token

Reply:
Yes, I added more information in the description. We are not forcing the user to use string value, but they can if they want. 
This is the same as any code that uses hugging face loader.

Reply:
for local run, expose the text token is fine. if it is passed into azureml jobs, I am not sure whether it has any security issues

Reply:
are there any existing requirements for exposing the text token? if not, I suggest to disable it from the time being, we can consider add it when we really have specific requirements in future.

Reply:
Done! removed the str typing for now.